x-pack/platform/plugins/private/upgrade_assistant/public/application/components/overview/upgrade_step/upgrade_step.tsx

```diff
@@ -145,6 +154,36 @@ const UpgradeStep = () => {
 
       <EuiSpacer size="m" />
 
+      <EuiCallOut title={i18nTexts.securitySolnLogsDBCalloutTitle}>
+        <FormattedMessage
+          id="xpack.upgradeAssistant.overview.securitySolutionLogsDBCalloutBody"
+          defaultMessage="Elastic Security users are advised to opt out of enabling logsdb index mode during this upgrade. Users should fully understand and accept the impact of using logsdb index mode with Elastic Security ({logsDataStreamDocs}), and ensure their deployment has excess hot data tier CPU resource capacity before enabling logsdb index mode, as it requires additional CPU resources during the data ingest/indexing process. Enabling logsdb index mode without sufficient hot data tier CPU resources may result in data ingestion backups and/or security detection rule timeouts and errors. Users who wish to use logsdb index mode and logs data streams may enable it after the upgrade ({indexModeWithSecuritySoln})."
```

Comment:
This feels a bit wordy, I took the suggested text verbatim but just adapted the link text provided. Would be great to get copy review.